classes/WP_CLI/ProgressBar.php

```diff
@@ -64,7 +84,7 @@ public function tick() {
 		$this->progress_bar->tick();
 		$this->total_ticks++;
 
-		do_action( 'action_scheduler/progress_tick', $this->total_ticks );
+		do_action( 'action_scheduler_progress_tick', $this->total_ticks );
```

Comment:
I apologize, there was a mistake in my earlier recommendation (which is probably why it was so confusing).

```suggestion
		do_action( 'action_scheduler/progress_tick', $this->total_ticks ); // phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores
```

I'll apply the above: we cannot (or, should not) change the hook name, so simply ignoring the rule here makes the most sense.

Reply:
I implemented your suggestion in [the previous comment](https://github.com/woocommerce/action-scheduler/pull/1134#discussion_r1815791089)! 😆